### ISSUE TITLE
Vertically align search in top navbar with text

### ISSUE THREAD

**Issue by apreshill** [quarto-dev/quarto-cli#58]
With a navbar defined by:

https://github.com/pommevilla/incog-workbench/blob/cb6673c6647518e54d33f903fd851777f801a328/_quarto.yml#L17-L26

```
  navbar:
    search: true
    logo: images/RStudio-ball.svg
    right:
      - text: "Rstudio.com"
        url: 'https://www.rstudio.com/'
      - text: "Release Notes"
        url: "https://support.rstudio.com/hc/en-us/articles/200716783-RStudio-Release-History"
      - text: "Help"
        url: "https://support.rstudio.com/hc/en-us"
```

We see: 
<img width="1436" alt="Screen Shot 2021-07-19 at 2 56 25 PM" src="https://user-images.githubusercontent.com/12160301/126233141-e046582f-e0a1-40d9-80ed-21ad508249cb.png">

It would look more streamlined to have the text aligned vertically with the search box.

Related issue: https://github.com/rstudio/rstudio-pro/issues/2747

**Comment by dragonstyle** [comment]
This should be fixed in v0.22.3 (should be available in a few minutes). Thanks for reporting it!

**Comment by dragonstyle** [comment]
Oops actually in this build:

https://github.com/quarto-dev/quarto-cli/releases/tag/v0.2.26
